testproject/Assets/Scripts/ColourChanger.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ColourChanger : MonoBehaviour
```

Comment:
nitpick but `ColorChanger ` instead of `ColourChanger`? :)

Reply:
Always w/ the inferior spelling :P 
Will change.